tests/std/tests/P0896R4_views_empty/test.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <ranges>
+#include <string>
+
+using namespace std;
+
+struct not_constructible {
+    not_constructible()  = delete;
+    ~not_constructible() = delete;
+};
+
+template <class T>
+constexpr bool test_one_type() {
+    // validate type properties
+    using R = ranges::empty_view<T>;
+    static_assert(ranges::view<R> && ranges::contiguous_range<R> && ranges::sized_range<R> && ranges::common_range<R>);
+    static_assert(same_as<const R, decltype(views::empty<T>)>);

```

Comment:
Should include `<concepts>` for `same_as`.

Reply:
Per the Standard, `<ranges>` includes `<iterator>` includes `<concepts>`.